README.md

```diff
@@ -47,8 +47,45 @@ SDX-LC: Acts as a bridge between the SDX Controller and the OXPOs.
 git clone --branch main --recurse-submodules https://github.com/atlanticwave-sdx/sdx-continuous-development
 
 cd sdx-continuous-development/
+
+git push --set-upstream origin main
+
+```
+
+### Creating Submodules
+
+## Execute only If submodules are not created inside the data-plane directory, (skip the creating submodules steps if cloning the repository)
+
+
+```
+cd sdx-continuous-development/data-plane
+
+git submodule add -b main https://github.com/atlanticwave-sdx/sdx-controller container-sdx-controller/sdx-controller/
+
+git submodule add -b integration https://github.com/atlanticwave-sdx/sdx-lc container-sdx-lc/sdx-lc/
+
+git submodule add -b main https://github.com/atlanticwave-sdx/kytos-sdx-topology container-kytos-sdx-topology/
+
+git submodule add -b main https://github.com/atlanticwave-sdx/sdx-meican container-sdx-meican/sdx-meican/
+
+git submodule update --init --recursive --remote
+
+git config --file=.gitmodules -l
+
+git submodule set-branch -b integration data-plane/container-sdx-lc/sdx-lc
+
+git submodule set-branch -b luisdev data-plane/container-kytos-sdx-topology
```

Comment:
@lmarinve since we agree on using the main branch and sending pull requests to merge changes there, I believe you should create small and well structured pull requests for kytos-sdx-topology and use the main branch here.